test/CollectionTest_Cpp.cc

```diff
@@ -113,15 +113,9 @@ class CollectionTest_Cpp : public CBLTest_Cpp {
 #define NOT_DELETE_DEFAULT_COLLECTION
 
 TEST_CASE_METHOD(CollectionTest_Cpp, "C++ Default Collection", "[Collection]") {
-    Collection col = db.getDefaultCollection();
-    REQUIRE(col);
-    CHECK(col.name() == "_default");
-    CHECK(col.scopeName() == "_default");
-    
-    col = db.getCollection(kCBLDefaultCollectionName);
-    REQUIRE(col);
-    CHECK(col.name() == "_default");
-    CHECK(col.scopeName() == "_default");
+    REQUIRE(defaultCollection);
```

Comment:
No need to check if this is already checked when getting the default collection.